shared/templates/coreos_kernel_option/kubernetes.template

```diff
@@ -1,3 +1,4 @@
+{{%- if not ARG_NEGATE %}}
```

Comment:
I don't think we need this macro, the `backends` should do trick. Or do you still want to have this?

Reply:
nope, let me remove it.